### PR TITLE
Support external-url

### PR DESCRIPTION
**Pull request by mtanda**

### CODE REVIEW

**Comment by fabxc** [comment]
Thanks.

The `-server` flag already accepts a full URL, so we don't have to add an additional flag. The problem is that we parse that URL but then overwrite the `URL.Path` component rather than appending to it.

**Comment by mtanda** [comment]
@fabxc I fixed it. Thanks for advice.

File: client.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ func (c *Client) Query(expr string) (QueryResponse, error) {
 		return nil, err
 	}
 
-	u.Path = queryPath
+	u.Path += queryPath
```
Comment (fabxc) [inline client.go]: Let's do `u.Path = strings.TrimRight(u.Path, "/") + queryPath` to be a bit cleaner.

Comment (mtanda) [inline client.go]: Sorry, I should do that.

**Comment by fabxc** [comment]
Excellent, thanks. :+1:

**Pull request merged**